executor/select_into.go

```diff
@@ -82,6 +87,34 @@ func (s *SelectIntoExec) considerEncloseOpt(et types.EvalType) bool {
 		et == types.ETJson
 }
 
+func (s *SelectIntoExec) escapeField(f []byte) []byte {
+	if s.intoOpt.FieldsInfo.Escaped == 0 {
+		return f
+	}
+	s.escapeBuf = s.escapeBuf[:0]
+	escape := func() {
```

Comment:
Pls remove the single statement function.

Reply:
addressed